### ISSUE TITLE
[Mainnet] Stacks Reward Endpoint  not returning data

### ISSUE THREAD

**Issue by OmkarB** [blockstack/stacks-blockchain-api#440]
**Describe the bug**
Even though the stacks btc rewards have started, the rewards endpoint - both aggregate/per user address is not returning data. 

**To Reproduce**
1. [This ](https://stacks-node-api.mainnet.stacks.co/extended/v1/burnchain/rewards) returns an empty response.

**Expected behavior**
Expecting to see btc rewards returned by the endpoint.

**Comment by zone117x** [comment]
This was caused by the wrong property name used for parsing burn chain reward messages. It has actually already been fixed here https://github.com/blockstack/stacks-blockchain-api/commit/a7f8a370f35d9bf1c922bedddee94c0937bf5e6f#r47136895

Once API version 0.47.5 is deployed (should be today 🤞), this issue should be resolved. I've tested running locally (only up to block ~2500 so far) and endpoints are working as expected. Also confirmed that the production pg tables for this data are empty right now.
